bindings/python/CMakeLists.txt

```diff
@@ -70,57 +39,29 @@ ENDIF(BUILD_WITH_LIBPYTHON)
 # Python exposition of FCL
 IF(NOT BUILD_WITH_HPP_FCL_PYTHON_BINDINGS)
   LIST(REMOVE_ITEM ${PROJECT_NAME}_PYTHON_HEADERS
-    multibody/fcl/transform.hpp
+    ${SOURCE_HEADERS_ROOT}/multibody/fcl/transform.hpp
     )
   LIST(REMOVE_ITEM ${PROJECT_NAME}_PYTHON_SOURCES
     multibody/fcl/expose-fcl.cpp
     )
 ENDIF(NOT BUILD_WITH_HPP_FCL_PYTHON_BINDINGS)
 
 IF(NOT BUILD_WITH_OPENMP_SUPPORT)
-  LIST_FILTER("${${PROJECT_NAME}_PYTHON_HEADERS}" "^multibody/pool" ${PROJECT_NAME}_PYTHON_HEADERS)
```

Comment:
@nim65s I don’t understand why we use LIST_FILTER instead of LIST(FILTER ...) ?

LIST(FILTER ...) exists since CMake 3.6, and our minimal cmake version is the 3.10.

Reply:
Because not so long ago we required CMake 3.1